configuration/analyzer_config.json

```diff
@@ -3275,6 +3275,23 @@
     },
     "params": {}
   },
+ "Whoisripe" : {
```

Comment:
rename to `WhoIs_RipeDB_Search` and change in Usage.md docs.

Reply:
Noted